src/db/file.rs

```diff
@@ -352,4 +332,17 @@ mod test {
         let files = get_file_list(env::current_dir().unwrap().join("Cargo.toml")).unwrap();
         assert_eq!(files[0], std::path::Path::new("Cargo.toml"));
     }
+
+    #[test]
+    fn test_mime_types() {
```

Comment:
Have you tested this with the old `magic` library as well?

Reply:
No